test/PgSubscription.js

```diff
@@ -298,6 +298,16 @@ function(test, done){
 
     Meteor.setTimeout(function() {
       test.equal(players.length, expectedRows.length);
+
+      // Test multiple quick changes to subscription
```

Comment:
I'll do that. And also the error occurs on the `registerStore`-method so I'll take a look if I can create an actual failing test.